games-arcade/wormux/wormux-20130927.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="A free Worms clone"
+DESCRIPTION="War has been declared! Battle your favorite free software mascots \  
+in the Wormux arena. With big sticks of dynamite, grenades, baseball bats, and \ 
+bazookas, you can exterminate your opponent in a 2D cartoon style scenery. \ 
+The goal of the game is to destroy all of your opponents' mascots."
+ 
+HOMEPAGE="https://github.com/a-team/wormux"
+COPYRIGHT="<a-team>"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="e0dc503460055a14a0d7fd94e66d7484264ee62b"
+SOURCE_URI="https://github.com/a-team/wormux/archive/$srcGitRev.tar.gz"
+SOURCE_FILENAME="wormux-$portVersion-$srcGitRev.tar.gz"
+CHECKSUM_SHA256="d5e66808de2e4912a64b583e3be4b8ea52cd220b8617a113507cca2c2291c6d3"
+SOURCE_DIR="wormux-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	wormux$secondaryArchSuffix = $portVersion
+"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGLU$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libsdl$secondaryArchSuffix
+	lib:libSDL_gfx$secondaryArchSuffix
+	lib:libSDL_image$secondaryArchSuffix
+	lib:libSDL_mixer$secondaryArchSuffix
+	lib:libSDL_net$secondaryArchSuffix
+	lib:libSDL_sound$secondaryArchSuffix
+	lib:libSDL_ttf$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libglu$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libsdl$secondaryArchSuffix
+	devel:libsdl_gfx$secondaryArchSuffix
+	devel:libsdl_image$secondaryArchSuffix
+	devel:libsdl_mixer$secondaryArchSuffix
+	devel:libsdl_net$secondaryArchSuffix
+	devel:libsdl_sound$secondaryArchSuffix
+	devel:libsdl_ttf$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:find
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix	
+	"
+
+BUILD()
+{
+#	./autogen.sh
+	runConfigure ./configure --disable-dependency-tracking --build=x86 
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+}
+	
+TEST()
+{
+	make check
+}
```

Comment:
If you have `/bin/ed`:
```bash
echo wq | ed wormux-20130927.recipe
```
This will add a line feed (i.e. character 0x0a) because it is currently missing here.